docs/source/notebooks/demo_models.py

```diff
@@ -0,0 +1,154 @@
+"""
```

Comment:
The license should look like: https://github.com/awslabs/graphstorm/blob/main/tests/end2end-tests/data_process/test_custom_mask_data.py#L1-L16